WordPress/Classes/System/WordPressAppDelegate.m

```diff
@@ -1559,4 +1568,36 @@ - (void)didReceiveNotificationCount:(NSInteger)count
     // Note: Empty method, just so silence compiler warning.
 }
 
+#pragma mark - What's new
+
+/**
+ *  @brief      Shows the What's New popup if needed.
+ *  @details    Takes care of saving the user defaults that signal that What's New was already
+ *              shown.  Also adds a slight delay before showing anything.  Also does nothing if
+ *              the user is not logged in.
+ */
+- (void)showWhatsNewIfNeeded
+{
+    if (![self noBlogsAndNoWordPressDotComAccount]) {
+        
+        static NSString* const WhatsNewUserDefaultsKey = @"WhatsNewUserDefaultsKey";
+        static const CGFloat WhatsNewShowDelay = 1.0f;
```

Comment:
Well, for static constants that are only used in a single method, and that really have no use elsewhere, I usually try to just keep it inside the method because then it means that if we remove the method, the constant will be cleaned up as well.

Please let me know if that sounds good or if you still prefer that outside of the method, and I'll take care of it.